src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -96,5 +89,32 @@
             </ul>
         </div>
     </div>
-    <tif:feedbackSessionPublishButton publishButton="${actions.publishButton}" buttonType="btn-default btn-xs margin-bottom-7px" />
-</c:if>
+    <div title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTS %>"
+         data-toggle="tooltip"
+         data-placement="top"
+         style="display: inline-block; padding-right: 5px;">
+        <div class="btn-group margin-bottom-2px">
```

Comment:
For the styling, I would think it would be better to follow that of the remind actions. `margin-bottom-2px` is not a defined/used class so change that to `margin-bottom-7px`?